gitkraken-client/insights.md

```diff
@@ -0,0 +1,44 @@
+---
+
+title: Insights
+description: GitKraken Insights is a powerful tool that helps you visualize how pull requests are merged into your repositories.
+taxonomy:
+    category: gitkraken-client
+
+---
+
+### What is GitKraken Insights?
+
+GitKraken Insights is a powerful tool that helps you visualize how pull requests are merged into your repositories. It provides a visual representation of your repository's history, allowing you to see how your codebase has evolved over time. You can use this information to make informed decisions about how to improve your workflow.
+
+<img src="/wp-content/uploads/insights.png" class="img-responsive center img-bordered">
+
+<div class='callout callout--warning'>
+    <p>
+        <strong>Note:</strong> Insights is in Preview mode, and we'd love to hear your thoughts and feedback. Just click on the "Provide feedback on this view" prompt in the upper left of the Pull Requests page to tell us what you think.
+    </p>
+</div>
+
+### How do I use GitKraken Insights?
+
+To enable GitKraken Insights, you’ll first need to open a Cloud Workspace and then navigate to the Pull Request section. From here, click to connect to your remote hosting service.
```

Comment:
As this message can be dismissed, we should also mention that you can connect insights for the desired service from Preferences > Integrations.